src/graphql.rs

```diff
@@ -1,10 +1,49 @@
 use goose::goose::{GooseUser, TransactionResult};
-use urlencoding::encode;
 
-pub async fn graphql_query_advisory(user: &mut GooseUser) -> TransactionResult {
-    let query = r#"{"query": "{getAdvisories { id sha256 published organization { id website } vulnerabilities { id title }}}" }"#;
-    let encoded_query = encode(query);
-    let url = format!("/graphql?query={}", encoded_query);
-    let _response = user.get(&url).await?;
+pub async fn g_get_advisories(user: &mut GooseUser) -> TransactionResult {
+    let body = r#"{"query": "{getAdvisories { id sha256 published organization { id website } vulnerabilities { id title }}}" }"#;
+    let _response = user.post("/graphql", body).await?;
+    Ok(())
+}
+
+pub async fn g_get_advisory_by_id(user: &mut GooseUser) -> TransactionResult {
+    let body = r#"{ "query": "{ getAdvisoryById(id: \"37292820-00ee-4299-8097-a11f8348bdf8\") { id title }}" }"#;
```

Comment:
Is this intentionally a negative test eg. I would have thought **uuid** is not guaranteed ?

Reply:
I see your point here, but I'm assuming the dump.sql content will continue the same as we have a step-by-step doc for this.